src/commands/mod.rs

```diff
@@ -906,6 +907,7 @@ impl DaemonControl {
             match create_spend(
                 &self.config.main_descriptor,
                 &self.secp,
+                true, // TODO: allow RBF of recovery transaction
```

Comment:
Heh, good point.

Reply:
The explicit parameter has now been removed, but I've added the TODO here: https://github.com/wizardsardine/liana/issues/853#issuecomment-2181096159.